code/modules/randomMaps/vaults.dm

```diff
@@ -14,12 +14,13 @@ var/const/vault_map_directory = "maps/randomVaults/"
 
 
 var/list/vault_map_names = list( //Add your vaults' map names to this list. Don't include the .dmm prefix
-	"icetruck_crash",
-	"asteroid_temple",
+	//"icetruck_crash",
+	//"asteroid_temple",
 	//"doomed_satelite", WARNING: this map is possessed. Uncommenting will cause bugs and general spookiness. Don't uncomment
-	"hivebot_factory",
-	"clown_base",
-	"rust"
+	//"hivebot_factory",
```

Comment:
Commenting these out is for testing purposes only to ensure I can get the vault to actually spawn.